report_qweb_parameter/demo/test_report_field_length.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="test_report_length_report_id" model="ir.actions.report">
+        <field name="name">Length Report</field>
+        <field name="model">res.company</field>
+        <field name="report_type">qweb-html</field>
+        <field name="report_name">report_qweb_parameter.test_report_length</field>
+    </record>
+    <template id="test_report_length">
+        <data>
+            <li
+                name="esc_length"
+                t-minlength="10"
+                t-length="10"
+                t-esc="docs[0].street"
```

Comment:
```suggestion
                t-out="docs[0].street"
```
`t-esc` and `t-raw` are deprecated output directives: https://www.odoo.com/documentation/18.0/developer/reference/frontend/qweb.html